Amoswachira/IRepoterv3ch4
# [#163221112]added more tests with cypress

### What does this PR do?
Adds minimal tests of the UI using cypress

### Description of Task to be completed?
Test the UI using the cypress framework

### How should this be manually tested?
clone the repo and navigate to the gh-pages branch and run 
```sh
node_modules/.bin/cypress open
```  
in the terminal Cypress will launch its own UI and then you’ll see the browser going through every step, as listed in your spec file.

### What are the relevant pivotal tracker stories?
#163221112
### Screenshots
![cypress](https://user-images.githubusercontent.com/17140636/51474927-efabfc00-1d91-11e9-9bdf-e80af3b43a8e.JPG)
